libnetwork/iptables/firewalld.go

```diff
@@ -149,9 +165,15 @@ func connectionLost() {
 
 // call all callbacks
 func reloaded() {
+	firewalldReloadMu.Lock()
+	defer firewalldReloadMu.Unlock()
 	for _, pf := range onReloaded {
 		(*pf)()
 	}
+
+	firewalldReloadedAtMu.Lock()
+	defer firewalldReloadedAtMu.Unlock()
+	firewalldReloadedAt = time.Now()
```

Comment:
I was wondering if we  should use `time.Now().UTC()` here, but it looks like we don't do so for `SystemTime` (wondering if we should 🤔)

https://github.com/moby/moby/blob/26e516dff4384bf3bc2233020a2f5f4001755efb/daemon/info.go#L58


```suggestion
	firewalldReloadedAt.Store(time.Now())
```

Reply:
Done.